packages/mdc-layout-grid/_mixins.scss

```diff
@@ -122,3 +122,17 @@
     align-self: stretch;
   }
 }
+
+@mixin mdc-layout-grid-fixed-column-width($size, $margin, $gutter, $column-width) {
```

Comment:
Can we also add the same error handling in the sass as we do in `mdc-theme`? e.g.:

```css
  @if not map-has-key($mdc-layout-grid-columns, $size) {
    @error "Invalid style specified! Choose one of #{map-keys($mdc-layout-grid-columns)}";
  }
```